recipes-config/systemd-resource-control/systemd-resource-control.bb

```diff
@@ -0,0 +1,87 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=3e927855e47b22eecd81954b0f545a0f"
+
+SRC_URI += " \
+    file://LICENSE \
+    file://audio.slice \
+    file://connectivity.slice \
+    file://coreservices.slice \
+    file://dlt.slice \
+    file://media.slice \
+    file://phone.slice \
+    file://speech.slice \
+    file://swdl.slice \
+    file://ui-apps.slice \
+    file://ui.slice \
+    file://vehiclefunctions.slice \
+    file://video.slice \
+    file://service.connectivity-manager.conf \
+"
+
+S = "${WORKDIR}"
+
+inherit allarch
+
+do_install_append() {
+    install -d ${D}${systemd_unitdir}/system/
+
+    # Slices
+    install -d ${D}${systemd_unitdir}/system/audio.slice.d/
+    install -m 0644 ${WORKDIR}/slice.audio.conf ${D}${systemd_unitdir}/system/audio.slice
+
+    install -d ${D}${systemd_unitdir}/system/connectivity.slice.d/
```

Comment:
delete row since we don't have any slice overrides for connectivity at the moment.